lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -373,6 +401,63 @@ protected function getReportbackInfo() {
 
 
   /**
+   * Collect data for third party services.
+   *
+   * @return array
+   */
+  protected function getServices() {
+    return [
+      'mobile_commons' => $this->getMobileCommonsData(),
+      'mailchimp' => $this->getMailChimpData(),
+    ];
+  }
+
+
+  /**
+   * Get the Scholarship amount for campaign if available.
+   *
+   * @return array|null
+   */
+  protected function getScholarship() {
+    return dosomething_helpers_extract_field_data($this->node->field_scholarship_amount);
+  }
+
+
+  /**
+   * Get the Solutions data for campaign if available; collects both the main solution copy
+   * and the solution support copy.
+   *
+   * @return array
+   */
+  protected function getSolutionData() {
+    $data['copy'] = dosomething_helpers_extract_field_data($this->node->field_solution_copy);
+    $data['support_copy'] = dosomething_helpers_extract_field_data($this->node->field_solution_support);
+
+    return $data;
+  }
+
+
+  /**
+   * Get status whether this campaign is a Staff Pick or not.
+   *
+   * @return bool
+   */
+  protected function getStaffPickStatus() {
+    return (bool) dosomething_helpers_extract_field_data($this->node->field_staff_pick);
```

Comment:
Where do you mean? I didn't touch any `Date()` in this update. I just moved around some methods to put it more in alphabetical order :) I'll note that above in description.